File: Tanker/Sources/TKRSharingOptions.m
#import "TKRSharingOptions.h"

@implementation TKRSharingOptions

+ (instancetype)options
{
  TKRSharingOptions* opts = [[self alloc] init];
  opts.shareWithUsers = @[];
  opts.shareWithGroups = @[];
  return opts;
}

@end

File: Tanker/Sources/TKRSharingOptions.swift
@objc(TKRSharingOptions)
public class SharingOptions: NSObject {
  @objc
  public var shareWithUsers: Array<String>;
  @objc
  public var shareWithGroups: Array<String>;

  @objc
  public override init() {
    self.shareWithUsers = [];
    self.shareWithGroups = [];
  }
}
